tests/integration_tests.rs

```diff
@@ -224,6 +224,16 @@ fn line_range_multiple() {
         .stdout("line 1\nline 2\nline 4\n");
 }
 
+#[test]
+fn line_range_increment() {
```

Comment:
I love that you added tests to the PR :+1:

Reply:
Thank you!!